### PR TITLE
DEV | Consolidate employee profile service

### PR DESCRIPTION
**Pull request by bseptember**
### Overview:

Removed EmployeeService and replaced with consolidated EmployeeProfileService

[Azure board](https://dev.azure.com/retro-rabbit/HRIS/_sprints/taskboard/HRIS%20Team/HRIS/Sprint%2026?System.AssignedTo=%40me&workitem=10184)

### CODE REVIEW

File: src/app/components/shared-components/store/effects/employeeprofile.effects.ts [hunk 1]
```diff
@@ -11,25 +11,26 @@ export class EmployeeProfileEffects {
...
+  // getEmployeeProfile$ = createEffect(() =>
+  //   this.actions$.pipe(
+      // ofType(EmployeeProfile.GetEmployeeProfile),
+      // exhaustMap(() =>
+        // this.employeeProfileService.GetEmployeeProfile().pipe(
+        //   map(employee => EmployeeProfile.GetEmployeeProfileSuccess({ employeeProfile: employee })),
+        // )
+  //     )
+  //   )
+  // );
```
Comment (madwizzard) [inline src/app/components/shared-components/store/effects/employeeprofile.effects.ts:23]: Is this supposed to be commented out?

File: src/app/components/shared-components/store/effects/employeeprofile.effects.ts [hunk 2]
```diff
@@ -11,25 +11,26 @@ export class EmployeeProfileEffects {
...
+  //   this.actions$.pipe(
+  //     ofType(EmployeeProfile.UpdateEmployeeProfile),
+  //     exhaustMap((UpdatedProfile: any) =>
+  //       this.employeeProfileService.UpdateEmployeeProfile(UpdatedProfile).pipe(
+  //         map(response => EmployeeProfile.UpdateEmployeeProfileSuccess({ response })),
+  //       )
+  //     )
+  //   )
+  // );
+
```
Comment (madwizzard) [inline src/app/components/shared-components/store/effects/employeeprofile.effects.ts:35]: Is this supposed to be commented out?

File: src/app/components/shared-components/store/effects/employeeprofile.effects.ts (hunk 1 above)
Comment (bseptember) [reply src/app/components/shared-components/store/effects/employeeprofile.effects.ts:23]: yes

File: src/app/components/shared-components/store/effects/employeeprofile.effects.ts (hunk 2 above)
Comment (bseptember) [reply src/app/components/shared-components/store/effects/employeeprofile.effects.ts:35]: yip

File: src/app/components/shared-components/store/effects/employeeprofile.effects.ts (hunk 1 above)
Comment (EduanO) [reply src/app/components/shared-components/store/effects/employeeprofile.effects.ts:23]: Should this not be deleted then?

**Review by EduanO — DISMISSED** [review]

**Review by Juanro-Minne — APPROVED** [review]

**Review by madwizzard — APPROVED** [review]

**Review by madwizzard — APPROVED** [review]

**Review by Juanro-Minne — APPROVED** [review]

**Review by EduanO — APPROVED** [review]

**Pull request merged**